positional_encodings/tf_positional_encodings.py

```diff
@@ -1,13 +1,26 @@
 import tensorflow as tf
 import numpy as np
 
-
 class TFPositionalEncoding2D(tf.keras.layers.Layer):
-    def __init__(self, channels, dtype=tf.float32):
+    def __init__(self, channels:int, return_format:str="sum", dtype=tf.float32):
```

Comment:
Should we make the default `pos` to ensure that the default behavior is the same across all modules?

Reply:
Or this could be something that we could change in an upcoming release for all the modules, including the previous pytorch ones.

Reply:
Yeah I think that it would be good to have consistency. I was also not sure about this one. I thought that most people would use it in "sum" mode. But both fine by me.